assets/scripts/lobby9.js

```diff
@@ -1848,8 +1848,20 @@ function scaleMiddleBoxes(){
     // $("#missionsBox").css("transform", "translateX(-50%) scale(" + ratioToReduce + ")")
     // $("#missionsBox").css("transform-origin", "bottom");
     $("#missionsBox").css("transform", "translateX(-50%) scale(" + ratioToReduce + ")");
-
-
+    var playerDivHeightRatio = $(".playerDiv").height()/128; 
+    var proposedTeamIcon = docCookies.getItem("optionDisplayProposedTeamIcon");
+    if(proposedTeamIcon === "shield"){
+        var maxHeight = 51;
+        var maxWidth = 40;
+    } else {
+        var maxHeight = 43;
+        var maxWidth = 122;
```

Comment:
Please declare these variables once, outside the if and else statements. Javascript ES2017 specifies that var has "function scope", so while this indeed works, its probably still better to have the variables declared once and assigned where needed inside if/else.